gp/fitness.py

```
@@ -0,0 +1,60 @@
+import numpy as np
+from deap import gp
+from . import utils
```

Ruff (0.11.9)

`.utils` imported but unused

Remove unused import: `.utils`

(F401)